scripts/plugin_setup/wordpress/setup_w3tc.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -ex
+
+scriptName="$0"
+redis_host="$1"
+redis_port="$2"
+redis_password="$3"
+wp_directory="$4"
+
+# function to print usage of this script to out
+function usage {
+    echo "usage: $scriptName <redis_host> <redis_port> <redis_password> <wp_directory>"
+    echo "      <redis_host>            : host url where redis cache is hosted"
+    echo "      <redis_port>            : port to connect on with redis"
+    echo "      <redis_password>        : password to authenticate to redis"
+    echo "      <wp_directory>          : path to wp directory"
+}
+
+# check if valid parameters are passed
+# check number of arguments passed
+if [[ $# -ne 4 ]]; then                                              
+    usage
+# arguments are not empty  
+elif [[ -z "$redis_host" || -z "$redis_password" || -z "$redis_port" || -z "$wp_directory" ]]; then            
+    usage
+fi
+
+redis_endpoint="$redis_host:$redis_port"
+w3tc_config_path="$wp_directory/wp-content/w3tc-config/master.php"
+
```

Comment:
can we check if the master.php file path is valid or not here?

Reply:
done, thanks